doc-content/drools-docs/src/main/asciidoc/DecisionEngine/cep-events-con.adoc

```diff
@@ -0,0 +1,18 @@
+[id='cep-events-con_{context}']
+
+= Events in complex event processing
+
+In {PRODUCT}, an event is a record of a significant change of state in the application domain. Depending on how the domain is modeled, the change of state may be represented by a single event, multiple atomic events, or hierarchies of correlated events. From a complex event processing (CEP) perspective, an event is a type of fact or object and a business rule is a definition of how to react to the data from that fact or object. For example, in a stock broker application, a change in security prices, a change in ownership from seller to buyer, or a change in an account holder's balance are all considered to be events because a change has occurred in the state of the application domain.
```

Comment:
This is a repetition of what written here https://github.com/sterobin/kie-docs/pull/31/files#diff-e9b8336c7b9a0f358500f0b06df9294cR5

Reply:
That's right. It's intentional. They overlap. And from a module file standpoint, each file needs to be standalone. Plus we can't assume the user is reading this in order, but make be jumping around. I've updated it too per my previous comment.